Emby.Server.Implementations/Library/LibraryManager.cs

```diff
@@ -392,9 +392,9 @@ public void DeleteItem(BaseItem item, DeleteOptions options, BaseItem parent, bo
                 // Add this flag to GetDeletePaths if required in the future
                 var isRequiredForDelete = true;
 
-                foreach (var fileSystemInfo in item.GetDeletePaths().ToList())
+                 foreach (var fileSystemInfo in item.GetDeletePaths())
```

Comment:
```suggestion
                foreach (var fileSystemInfo in item.GetDeletePaths())
```
No idea how this happend